packages/nuxt/src/core/nitro.ts

```diff
@@ -249,7 +249,8 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
     const buildId = nuxt.options.runtimeConfig.app.buildId ||= nuxt.options.buildId
     const buildTimestamp = Date.now()
 
-    const manifestPrefix = joinURL(nuxt.options.app.buildAssetsDir, 'builds')
+    const baseURL = nuxt.options.runtimeConfig.app.baseURL ||= nuxt.options.app.baseURL
```

Comment:
Not sure if we do need the reassign like we do in L249 for buildId 🤔

`const buildId = nuxt.options.runtimeConfig.app.buildId ||= nuxt.options.buildId`

Reply:
we do not need to reassign

Reply:
Okay, will remove it then

Reply:
i think we don't need to check runtimeConfig here at all